Phoenix/PHShebangPreprocessor.m

```diff
@@ -74,10 +74,24 @@ + (NSString *) process:(NSString *)script atPath:(NSString *)path error:(NSError
         return script;
     }
 
-    // Read past shebang-directive
-    [standardOutputFile readDataOfLength:scanner.scanLocation];
+    // Read past shebang-directive (hold onto it incase it's not the shebang text)
```

Comment:
Didn't think it was really worth running another scan, when we can easily catch the shebang length substring, and join them again if required.

We could also just grab `availableData`